generators/static-api/templates/_package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@springworks/<%=appname%>-static",
+  "version": "0.0.1",
+  "description": "Static implementation of the API",
+  "main": "index.js",
+  "bin": {
+    "start-<%=appname%>-static": "bin/start-server.js",
+    "stop-<%=appname%>-static": "bin/stop-server.js"
+  },
+  "engines": {
+    "node": ">=4.2.2",
+    "npm": ">=3.3.12"
+  },
+  "dependencies": {
+    "@springworks/static-api-server": "1.0.2",
+    "fixture-loader": "0.2.5",
+    "forever": "0.15.1"
```

Comment:
- [ ] Remove "dependencies" object and install dependencies in the class in index.js (see other comment)